README.md

```diff
@@ -25,7 +25,14 @@ steps.forEach(step => {
 
 To solve an equation:
 ```js
-const steps = mathsteps.solveEquation('2x + 2x + x + x');
+const steps = mathsteps.solveEquation('2x + 3x = 35');
+
+steps.forEach(step => {
+    console.log("before change: " + step.oldEquation.print());  // e.g. before change: 2x + 3x = 35
```

Comment:
@evykassirer But the earlier example for `simplifyExpression` also has comments on the same lin as the code and it leads to 92 characters in line 19!